official/benchmark/models/resnet_imagenet_main.py

```diff
@@ -37,6 +38,40 @@
 from official.vision.image_classification.resnet import resnet_model
 
 
+def cluster_last_three_conv2d_layers(model):
+  last_three_conv2d_layers =  [
+      layer for layer in model.layers
+      if isinstance(layer, tf.keras.layers.Conv2D) and
+      not isinstance(layer, tf.keras.layers.DepthwiseConv2D)
```

Comment:
filter for `DepthwiseConv2D` not needed anymore. weight clustering skips it now.